sagemaker-pipelines/time_series_forecasting/amazon_forecast_pipeline/README.md

```diff
@@ -0,0 +1,16 @@
+# Amazon Forecast with SageMaker Pipelines
+
+This SageMaker example showcases how you can create a dataset, dataset group and predictor with Amazon Forecast and SageMaker Pipelines.
+
+## Contents
+
+[sm_pipeline_with_amazon_forecast.ipynb](sm_pipeline_with_amazon_forecast.ipynb): Notebook explaining the pipeline step-by-step.
+
+[preprocess.py](preprocess.py): Script used in the ForecastPreProcess step in pipeline for data preparation used for training and evaluation
```

Comment:
Grammar nit: end this sentence with a period to be consistent (and the "data" line).

Optionally wrap the step names in backticks for code formatting (such as `ForecastPreProcess`)